main.go

```
@@ -265,9 +265,7 @@ func PostImage(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	fmt.Fprintf(w, "Successfully saved: %s", filePath)
-
-	w.Write([]byte(filepath.Join(Config.Domain, filePath)))
+	w.Write([]byte(filepath.Join(Config.Domain, folder, id+"."+format)))
```

golangci-lint

Error return value of `w.Write` is not checked

(errcheck)